Library/Formula/bwctl.rb

```diff
@@ -16,15 +16,113 @@ class Bwctl < Formula
   depends_on "iperf3" => :optional
   depends_on "thrulay" => :optional
 
+  patch :DATA
+
   def install
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--mandir=#{man}"
     system "make", "install"
+
+    (buildpath+"bwctld.conf").write bwctld_conf
+    (buildpath+"bwctld.keys").write ""
+    (buildpath+"bwctld.limits").write bwctld_limits
+    etc.install "bwctld.conf"
+    etc.install "bwctld.keys"
+    etc.install "bwctld.limits"
+
+    (var+"log/bwctld").mkpath
+  end
+
+  def bwctld_conf; <<-EOS.undent
+    allow_unsync
+    log_location
+    peer_port       6001-6200
+    facility        local5
+    test_port       5001-5900
+    iperf_port      5001-5300
+    nuttcp_port     5301-5600
+    owamp_port      5601-5900
+    user            nobody
+    group           nobody
+    EOS
+  end
+
+  def bwctld_limits; <<-EOS.undent
+    limit root with allow_udp=on, bandwidth=900000000, max_time_error=20, allow_tcp=on, allow_open_mode=on, duration=60
+    limit regular with allow_udp=off, parent=root
+    limit jail with allow_udp=off, bandwidth=1, parent=root, allow_tcp=off, allow_open_mode=off, duration=1
+    assign net ::1/128 root
+    assign net 127.0.0.1/32 root
+    assign net 172.16.10.0/24 jail
+    assign default regular
+    EOS
+  end
```

Comment:
Also, how have you decided on these values and will they be suitable for all users/networks?